index.html

```diff
@@ -0,0 +1,339 @@
+              </li>
+              <li>
+                <ul class="languages-elements">
+                  <li>
+                    <span>
+                      <img
+                        class="Language-logo"
+                        src="Asset/JS.png"
+                        alt="JavaScript"
+                        width="70px"
+                        height="70px"
+                      />
+                    </span>
+                    <p>JavaScript</p>
+                  </li>
+                  <li>
+                    <span>
+                      <img
+                        class="Language-Logo"
+                        src="Asset/HTML5.png"
+                        alt="HTML"
+                      />
+                    </span>
+                    <p>HTML</p>
+                  </li>
+                  <li>
+                    <span>
+                      <img
+                        class="Language-Logo"
+                        src="Asset/CSS3.png"
+                        alt="CSS"
+                      />
+                    </span>
+                    <p>CSS</p>
+                  </li>
+                </ul>
+              </li>
+              <li class="skill-title Skill-border">
+                <h3 class="skill-heading">Frameworks</h3>
+                <span class="listIcon"
+                  ><img src="Asset/icon.png" alt="List Skils" />
+                </span>
+              </li>
+              <li class="skill-title Skill-border">
+                <h3 class="skill-heading">Skills</h3>
+                <span class="listIcon"
+                  ><img src="Asset/icon.png" alt="List Skils" />
+                </span>
+              </li>
+            </ul>
+          </div>
+        </section>
+        <div class="contact-MeForm">
+          <section class="contact-application" id="contact">
+            <div class="form-wrapper">
+              <h2 class="contact-me">Contact me</h2>
+              <p class="ContactDesctiption">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it!
+              </p>
+              <form id="contact-form" action="" method="POST">
+                <div class="form-group">
+                  <input
+                    type="text"
+                    id="name"
+                    name="name"
+                    maxlength="30"
+                    placeholder="Melkamu Almawu"
+                    required
+                  />
+                </div>
+                <div class="form-group">
+                  <input
+                    type="email"
+                    id="email"
+                    name="email"
+                    placeholder="malmawu@gmail.com"
+                    required
+                  />
+                </div>
+                <div class="form-group">
+                  <textarea
```

Comment:
- [ ] Kindly fix the following sections.

- Figma design
<img width="281" alt="image" src="https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/41163571/a342541d-6c95-40d8-af87-fc399e0bba10">

- Your design
<img width="286" alt="image" src="https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/41163571/684499f0-3d16-4d31-9c12-c3bb43af1e75">


- The margin-bottom of the contact me heading.
- The stretching of the text underneath the heading
- The margin bottom of the text.
- Center the Get in touch button.
- Reduce the strectch of the content a bit.